os-config.tpl.yml

```diff
@@ -146,6 +146,7 @@ rancher:
       - /usr/bin/ros:/usr/bin/cloud-init-save:ro
       - /usr/bin/ros:/usr/sbin/netconf:ro
       - /usr/bin/ros:/usr/sbin/wait-for-docker:ro
+      - /usr/rancher:/usr/share/rancher:ro
```

Comment:
@joshwget we use `rancher` everywhere else, so I'm going for consistent - we can revisit that in a few weeks when/if we have a rancher agent/server service